proto/anki/deckconfig.proto

```diff
@@ -32,22 +32,33 @@ message DeckConfig {
       NEW_CARD_INSERT_ORDER_RANDOM = 1;
     }
     enum NewCardGatherPriority {
+      // Decks in alphabetical order (preorder), then ascending position.
+      // Siblings are consecutive, provided they have the same position.
       NEW_CARD_GATHER_PRIORITY_DECK = 0;
+      // Ascending position.
+      // Siblings are consecutive, provided they have the same position.
       NEW_CARD_GATHER_PRIORITY_LOWEST_POSITION = 1;
+      // Descending position.
+      // Siblings are consecutive, provided they have the same position.
       NEW_CARD_GATHER_PRIORITY_HIGHEST_POSITION = 2;
-      NEW_CARD_GATHER_PRIORITY_RANDOM = 3;
+      // Siblings are consecutive.
+      NEW_CARD_GATHER_PRIORITY_RANDOM_NOTES = 3;
+      // Siblings are neither grouped nor ordered.
+      NEW_CARD_GATHER_PRIORITY_RANDOM_CARDS = 4;
     }
     enum NewCardSortOrder {
-      NEW_CARD_SORT_ORDER_TEMPLATE_THEN_LOWEST_POSITION = 0;
-      NEW_CARD_SORT_ORDER_TEMPLATE_THEN_HIGHEST_POSITION = 1;
+      // Ascending card template ordinal.
+      // For a given ordinal, cards appear in gather order.
+      NEW_CARD_SORT_ORDER_TEMPLATE = 0;
+      // Preserves original gather order (eg deck order).
+      NEW_CARD_SORT_ORDER_NO_SORT = 1;
+      // Ascending card template ordinal.
+      // For a given ordinal, cards appear in random order.
       NEW_CARD_SORT_ORDER_TEMPLATE_THEN_RANDOM = 2;
-      NEW_CARD_SORT_ORDER_LOWEST_POSITION = 3;
-      NEW_CARD_SORT_ORDER_HIGHEST_POSITION = 4;
-      NEW_CARD_SORT_ORDER_RANDOM = 5;
-      // Sorts by template, preserving original gather order.
-      NEW_CARD_SORT_ORDER_TEMPLATE_ONLY = 6;
-      // Preserves original gather order (eg deck order)
-      NEW_CARD_SORT_ORDER_NO_SORT = 7;
```

Comment:
If the user has selected something like option 7 in a previous Anki version, the selection box will display as blank in our deck options.  One way to work around it would be something like the following inside the $: { ... } of DisplayOrder.svelte, after making the changes mentioned in the previous comment:

        if (!($config.newCardSortOrder in SortOrder)) {
            $config.newCardSortOrder = 0;
        }

Reply:
(we'll probably at least also want to add a note to the change notes directing users to check their options after updating)